doc/source/reference/security.rst

```diff
@@ -0,0 +1,60 @@
+NumPy security considertions and reporting
+==========================================
+
+Security issues can be reported privately as described on the issue tracker.
```

Comment:
To make that easier to find, I propose: `as described under *Report a security vulnerability* in the project README and under [new issue in the issue tracker]()https://github.com/numpy/numpy/issues/new/choose).`

Reply:
(but then with a reST link format rather than Markdown)